### PR TITLE
Feature/pybind dl3

### PR DESCRIPTION
**Pull request by Shadowninja196**
Added a wrapper header for run_simple_service and updated various files to incorporate the new wrapper. Also updated some comments.

### CODE REVIEW

File: module_bindings/dripline_core/run_simple_service_pybind.hh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include "run_simple_service.hh"
+#include "pybind11/pybind11.h"
+
+namespace dripline_pybind {
```
Comment (laroque) [inline module_bindings/dripline_core/run_simple_service_pybind.hh]: our convention is that `{}` blocks are opened on the following line, with both at the same indentation as the line that creates the block, so:
```
namespace dripline_pybind
{
    <stuff>
}
```

File: module_bindings/dripline_core/run_simple_service_pybind.hh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  void export_run_simple_service(pybind11::module& mod){
```
Comment (laroque) [inline module_bindings/dripline_core/run_simple_service_pybind.hh]: same comment for `{}` blocks

our convention for arguments is to include a space after the open and before the close, so:
`void export_run_simple_service( pybind11::module& mod )`

File: module_bindings/dripline_core/run_simple_service_pybind.hh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    pybind11::class_<dripline::run_simple_service, std::shared_ptr<dripline::run_simple_service> >(mod, "run_simple_service")
```
Comment (laroque) [inline module_bindings/dripline_core/run_simple_service_pybind.hh]: same comment on `()`, also applies to template arguments `<>`

Comment (laroque) [reply module_bindings/dripline_core/run_simple_service_pybind.hh]: please also add a class doc string, should just be another string argument; I'm not sure how consistently we've applied them, but we would like to have them on basically everything we choose to bind.

File: module_bindings/dripline_core/run_simple_service_pybind.hh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+      .def(pybind11::init<const scarab::param_node&>())
```
Comment (laroque) [inline module_bindings/dripline_core/run_simple_service_pybind.hh]: spaces for `()` and `<>`, can skip doc string on `__init__` if it takes no arguments

File: module_bindings/dripline_core/run_simple_service_pybind.hh [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+      .def("execute", &dripline::run_simple_service::execute);
```
Comment (laroque) [inline module_bindings/dripline_core/run_simple_service_pybind.hh]: same comments: spaces and doc string

File: module_bindings/dripline_core/run_simple_service_pybind.hh (hunk 3 above)
Comment (Shadowninja196) [reply module_bindings/dripline_core/run_simple_service_pybind.hh]: So something like _void export_run_simple_service( pybind11::module& mod, std::string doc )_

**Pull request merged**